src/aes.rs

```diff
@@ -99,4 +146,24 @@ mod test {
         let result = substitute_16_bytes(num, cs);
         assert_eq!(u128::from_le_bytes(expected), result.0);
     }
+    #[test]
+    fn test_shift() {
+        let cs = ConstraintSystem::<Fq>::new_ref();
+        // This example is taken from this blog-post:
+        // https://towardsdatascience.com/aes-encryption-256-bit-a9ae49cde0b6
+        let expected: [u8; 16] = [
+            0xd4, 0xe0, 0xb8, 0x1e, 0xbf, 0xb4, 0x41, 0x27, 0x5d, 0x52, 0x11, 0x98, 0x30, 0xae,
+            0xf1, 0xe5,
+        ];
+        let value_to_shift: [u8; 16] = [
+            0xd4, 0xe0, 0xb8, 0x1e, 0x27, 0xbf, 0xb4, 0x41, 0x11, 0x98, 0x5d, 0x52, 0xae, 0xf1,
+            0xe5, 0x30,
+        ];
```

Comment:
```suggestion
        let value_to_shift: [u8; 16] = [
            0xd4, 0xe0, 0xb8, 0x1e, 0x27, 0xbf, 0xb4, 0x41, 0x11, 0x98, 0x5d, 0x52, 0xae, 0xf1,
            0xe5, 0x30,
        ];
        let expected: [u8; 16] = [
            value_to_shift[0], value_to_shift[5], value_to_shift[10], value_to_shift[15],
            value_to_shift[4], value_to_shift[9], value_to_shift[14], value_to_shift[3],
            value_to_shift[8], value_to_shift[13], value_to_shift[2], value_to_shift[7],
            value_to_shift[12], value_to_shift[1], value_to_shift[6], value_to_shift[11],
        ];
```

Following the AES ShiftRows step the `expected` shifted value (if we rotate once) should look like this.